src/types/module.rs

```diff
@@ -133,12 +151,31 @@ impl PyModule {
     /// let code = std::fs::read_to_string("assets/script.py")?;
     ///
     /// Python::with_gil(|py| -> PyResult<()> {
-    ///     PyModule::from_code_bound(py, &code, "example.py", "example")?;
+    ///     PyModule::from_code(py, CString::new(code)?.as_c_str(), c"example.py", c"example")?;
```

Comment:
Sadly we still need to wait a bit until we can use c-string literals

Reply:
I wonder why MSRV does not complain here 🤔

Reply:
Changed it to `c_str!` here as well